packages/ses/src/lockdown.js

```diff
@@ -188,11 +188,24 @@ export const repairIntrinsics = (options = {}) => {
       'safe',
     ),
     __hardenTaming__ = getenv('LOCKDOWN_HARDEN_TAMING', 'safe'),
-    dateTaming = 'safe', // deprecated
-    mathTaming = 'safe', // deprecated
+    dateTaming, // deprecated
+    mathTaming, // deprecated
     ...extraOptions
   } = options;
 
+  if (dateTaming !== undefined) {
+    // eslint-disable-next-line no-console
+    console.warn(
```

Comment:
After rebase, this will need to be `const { warn } = reporter; …; warn(…)` since we added a layer of indirection for `lockdown({ reporting: 'platform' | 'console' | 'none' })`.